reference/features/configuration.md

```diff
@@ -230,7 +230,7 @@ The `update` database handles [pending updates](/learn/advanced/asynchronous_upd
 **CLI option**: `--http-payload-size-limit`
 **Default value**: `104857600` (~100MB)
 
-Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+Sets the maximum size of [accepted payloads](https://docs.meilisearch.com/learn/core_concepts/documents.html#dataset-format). Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
```

Comment:
Very important thing here: always prefer relative links when linking to stuff within the docs website. I think the only place in the docs where we can't use relative links is on `errors.yaml`.

In case you need a refresher:

- absolute links specify the full path and protocol to a certain resource (e.g. an image, a page): `https://example.com`;
- relative links specify the path relative to the resource you're writing: `./an_image.jpg`. To make things easier, you can make them relative to the root of the website: `/learn/core_concepts/documents.md#dataset_format`. Note that we need to link to the `.md` file and not to the final compiled html resource.

There are a few reasons for this, but one of the most immediately useful for us is that our dead link checker only inspects relative links.

```suggestion
Sets the maximum size of [accepted payloads](/learn/core_concepts/documents.md#dataset-format). Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
```